### DISCUSSION TITLE
Mehrere KNX IP Gateways mit hkknx

### DISCUSSION THREAD

**Discussion by uwearzt** [brutella/hkknx-public#119] [General]
Hi,

hab die 2.0 Beta getestet, sieht sehr gut aus. Vor dem Kauf noch zwei Fragen:

* Wenn ich zwei KNX/IP Gateways betreibe (1 * Haus, 1 * Garten) mit getrennten KNX Netzen betreibe, kann ich mit einer Lizenz einfach zwei hkknx Instanzen starten und beide KNX Bridges im Homekit einbinden?
* Wie importiere ich das knxproj (das beide Teilnetze enthaelt) am besten um nur die relevanten Adressen in der Bridge zu haben?

Thanks
Uwe

**Comment by brutella** [comment]
> Wenn ich zwei KNX/IP Gateways betreibe (1 * Haus, 1 * Garten) mit getrennten KNX Netzen betreibe, kann ich mit einer Lizenz einfach zwei hkknx Instanzen starten und beide KNX Bridges im Homekit einbinden?

Ja, solange die Instanzen auf dem selben Computer laufen, können diese mit einer Lizenz betrieben werden.

> Wie importiere ich das knxproj (das beide Teilnetze enthaelt) am besten um nur die relevanten Adressen in der Bridge zu haben?

Importieren von spezifischen Gruppenadressen ist zurzeit nicht möglich. Du müsstest also wissen, welche Gruppenadressen für welchen Bereich sind.
